python/pyspark/loose_version.py

```diff
@@ -0,0 +1,69 @@
+import re
+
+
+class LooseVersion:
```

Comment:
Does this copy from `distutils`? If so, maybe we need to add a few comment here to explain it?

Reply:
For example, `python/docs/source/_static/copybutton.js` has few lines of comments.

Reply:
Actually, it's reimplemented by squashing `Version` class into the existing `LooseVersion` class. Let me add that.